fakeredis.py

```diff
@@ -547,6 +547,9 @@ def type(self, name):
             return b'list'
         elif isinstance(key, set):
             return b'set'
+        else:
+            assert key is None
```

Comment:
Should the assert be here?  Is it possible for key to be not be None at this point?